files/galaxy/tpv/tools.yml

```diff
@@ -422,6 +422,8 @@ tools:
     inherits: basic_docker_tool
   toolshed.g2.bx.psu.edu/repos/bgruening/biomodels_biomd0000001076/biomodels_biomd0000001076/.*:
     inherits: basic_docker_tool
+  toolshed.g2.bx.psu.edu/repos/bgruening/image_processing/yolo-utils/doclayoutyolo/doclayoutyolo/.*:
```

Comment:
add please the other YOLO tools as well, when they use docker, please check ...

Thanks

Reply:
I added them. I think `json2yolosegment` may not require GPUs. I have added it to use `basic container` but Yolo train and predict tools, I have added them with GPU